index.js

```diff
@@ -70,20 +71,23 @@ program
                 console.log(`Error ${resp.status}`);
                 return;
             }
-
             break;
         }
 
         console.log('Decoding...');
         contributions = conversion.decode(resp.contributions);
+        //contributions = conversion.decode('{"contributions": [ '+resp.contributions+'}');
 
-        const randValue = Math.floor(Math.random() * 100000); // TODO:
+        rand = contribute.generateRandom();
+        rand = Fr.create(rand);
```

Comment:
What this means?

Reply:
We can get a valid number if we put a number that satisfies fr.